tests/ecs.rs

```diff
@@ -0,0 +1,27 @@
+//#![cfg(feature = "ecs")]
```

Comment:
I think this needs to be uncommented to run the integration test with the --features flag to cargo.